octodns/provider/route53.py

```diff
@@ -230,7 +230,7 @@ class Route53Provider(BaseProvider):
 
     # This should be bumped when there are underlying changes made to the
     # health check config.
-    HEALTH_CHECK_VERSION = '0000'
+    HEALTH_CHECK_VERSION = '0001'
```

Comment:
Have to bump Route53's HC version, this will cause a replacement of all health-checks via _extra_changes including all Geo records in the plan.